connector-runtime/spring-boot-starter-camunda-connectors/pom.xml

```diff
@@ -27,14 +27,18 @@
     </dependency>
     <dependency>
       <groupId>io.camunda</groupId>
-      <artifactId>spring-boot-starter-camunda-sdk</artifactId>
+      <artifactId>zeebe-client-java</artifactId>
       <version>${version.zeebe}</version>
     </dependency>
```

Comment:
We might actually be able to remove this here as its already defined in the runtime-core.